newsfragments/2860.misc.rst

```diff
@@ -0,0 +1 @@
+Moved the metadata into :pep:`PEP 621`-compliant :file:`pyproject.toml`.
```

Comment:
Commenting from phone, I didn't fully clean up the previous suggested change. This should be better:
```suggestion
Moved the metadata into :pep:`621`-compliant :file:`pyproject.toml`.
```